ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectLoginTest.kt

```diff
@@ -252,21 +225,12 @@ class RedirectLoginTest : StringSpec() {
 
         "Empty Redirect AuthnRequest Test" {
             Log.debugWithSupplier { "Empty Redirect AuthnRequest Test" }
-            val authnRequest = AuthnRequestBuilder().buildObject().apply {
-            }
-
-            val authnRequestString = authnRequestToString(authnRequest)
-            Log.debugWithSupplier { authnRequestString.prettyPrintXml() }
-
-            val encodedRequest = Encoder.encodeRedirectMessage(authnRequestString)
+            val authnRequest = AuthnRequestBuilder().buildObject()
+            val encodedRequest = encodeAuthnRequest(authnRequest)
+            val queryParams = mutableMapOf<String, String>()
```

Comment:
✏️ Should be able to replace the mutable map/put with 
`val queryParams = mapOf(SAML_REQUEST to encodedRequest)`